iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -657,6 +657,11 @@ static LogicalResult legalizeDispatchWorkgroupOperands(
 /// order, i.e. starting from the outer-most to innermost.
 static SmallVector<unsigned> getPartitionedLoops(Operation *op) {
   SmallVector<unsigned> partitionedLoops;
+  if (auto mmt4dOp = dyn_cast<linalg::Mmt4DOp>(op)) {
```

Comment:
This is the right place to do this, but this might not work as you expect. This also needs to be respected during "initCPULaunchConfiguration`. To address this PR #6615  adds the same method to Codegen backend (https://github.com/google/iree/pull/6615/files#diff-4b5bfd1aaacfbfe7e6c424a57fc9fbec9b5eee74cd7e7f9e87734bffb43f57b4R43). Then both the dispatch region creation and the backends see the same information. To avoid "dependencies" this is just duplicated there, but let me try to find a common place for this.

Do you mind rebasing on top of that PR. There might be some small changes because of the PR (I'll add you as reviewer there to see the changes)

Reply:
Getting lots of merge conflicts when rebasing =), and yes I agree a common `getPartitionedLoops` is better. 
I will wait for your PR to land then update this.. thanks!

Reply:
I already landed. I can help you with merge conflicts. if you still have them.